core/src/main/scala/com/eharmony/spotz/optimizer/grid/GridSearchResult.scala

```diff
@@ -17,15 +17,7 @@ class GridSearchResult[P, L](
   extends OptimizerResult[P, L](bestPoint, bestLoss) {
 
   override def toString = {
```

Comment:
Is there any harm in overriding a case class' toString implementation?  I know I get it by default for free, but I'd prefer to use the toString I implemented.